VirtusFitConsole/Program.cs

```diff
@@ -24,14 +24,86 @@ static void Main(string[] args)
                 EditDataFromConsoleInterface testInterface = new EditDataFromConsoleInterface();
                 testInterface.EditProductInterface();
 
+                AddProductFromConsole();
+                DisplayProductList.DisplayList(ListOfProducts);
+                testInterface.EditProductInterface();
                 DisplayProductList.DisplayList(ListOfProducts);
             }
             catch (Exception e)
             {
                 Console.WriteLine($"An error occured: {e.Message} \nPress any key.");
                 Console.ReadKey();
             }
+        }
 
+        private static int ID = 9000;
+        public static void AddProductFromConsole()
+        {
+            try
+            {
+                Console.WriteLine("Product name:");
+                string productName = Console.ReadLine();
+                Console.WriteLine("Weight unit (G for gramms/ML for mililiters):");
+                string portionUnit = Console.ReadLine().ToUpper();
+                if (portionUnit != "G" && portionUnit !="ML")
```

Comment:
units should be lowercase "g" and "ml" to stay correct with SI unit system, we should change it in other files as well